From 465817d888c058446e7836ea0fe7721a803ce2c0 Mon Sep 17 00:00:00 2001
From: Sven Roederer <freifunk@it-solutions.geroedel.de>
Date: Thu, 5 Sep 2019 22:52:21 +0200
Subject: [PATCH] patches: hardcode to use python2 for building wireless-regdb

this avoids build-failues when hosts python is python3. The wireless-
regdb-code is still python2.
---
 .../openwrt/100-wireless-regdb_use_python2.patch    | 13 +++++++++++++
 1 files changed, 13 insertions(+)
 create mode 100644 patches/openwrt/100-wireless-regdb_use_python2.patch

diff --git a/package/firmware/wireless-regdb/Makefile b/package/firmware/wireless-regdb/Makefile
index d06da2c708..28234803ab 100644
--- a/package/firmware/wireless-regdb/Makefile
+++ b/package/firmware/wireless-regdb/Makefile
@@ -20,7 +20,7 @@ define Package/wireless-regdb
 endef
 
 define Build/Compile
-	python $(PKG_BUILD_DIR)/db2fw.py $(PKG_BUILD_DIR)/regulatory.db $(PKG_BUILD_DIR)/db.txt
+	python2 $(PKG_BUILD_DIR)/db2fw.py $(PKG_BUILD_DIR)/regulatory.db $(PKG_BUILD_DIR)/db.txt
 endef
 
 define Package/wireless-regdb/install
-- 
2.20.1

